src/docs/blogs/test.md

```diff
@@ -0,0 +1,9 @@
+# Title: This is a sample blog
+
+# Author: John
+
+# Date: 4.4.17
+
+### Content:
```

Comment:
OK. Also, could you send a sample userblog.md file? I am still confused about the format that the user will follow while writing the md file.